vendor/bat-native-confirmations/src/bat/confirmations/internal/time.cc

```diff
@@ -14,4 +14,19 @@ uint64_t Time::NowInSeconds() {
   return now.ToDoubleT();
 }
 
+uint64_t Time::MigrateToDoubleT(const uint64_t seconds) {
+  if (date_in_seconds < 10000000000) {
+    // Already migrated as DoubleT will never reach 10000000000 in our lifetime
+    return date_in_seconds;
+  }
+
+  // Migrate date to DoubleT
+  auto now = base::Time::Now();
+  auto now_in_seconds = static_cast<uint64_t>((now - base::Time()).InSeconds());
+  auto delta = seconds - now_in_seconds;
```

Comment:
Is delta meant to be negative or positive? It seems it will be negative, if we're subtracting `now_in_seconds` from it. Just wasn't sure if that's intended.

Reply:
This is intended. Delta could be negative or positive